src/lib.rs

```diff
@@ -1,7 +1,7 @@
 #![crate_name = "gdal"]
 #![crate_type = "lib"]
-// Enable `doc_cfg` features when `docsrs` is defined by docs.rs config
-#![cfg_attr(docsrs, feature(doc_cfg))]
+// Document all features on docs.rs
+#![cfg_attr(docsrs, feature(doc_auto_cfg))]
```

Comment:
Like this comment if you lost almost one hour because of `RUSTFLAGS` vs. `RUSTDOCFLAGS`.